README.md

```diff
@@ -139,7 +139,20 @@ models:
           compare_columns:
             - first_name
             - last_name
-    columns: ...
+```
+
+Alternatively, if you want to compare all columns apart from a few, you can use the `exclude_columns` field:
+
+```yaml
+models:
+  - name: stg_customers
+    tests:
+      - dbt_datamocktool.unit_test:
+          input_mapping:
+            source('jaffle_shop', 'raw_customers'): ref('dmt__raw_customers_1')
+          expected_output: ref('dmt__expected_stg_customers_1')
+          exclude_columns:
+            - description
```

Comment:
I'm getting an error when I run this (on Postgres):

```
14:58:47  Database Error in test dbt_datamocktool_unit_test_stg_customers_customer_id__first_name__last_name__ref_dmt__expected_stg_customers_1___ref_dmt__raw_customers_1_ (models/staging/schema.yml)
14:58:47    column "description" does not exist
14:58:47    LINE 72:     "description" 
```

I guess it's because `description` exists in the expectation seed but not in the original table. How do you want to handle this?

Reply:
Huh, I wonder why postgres kicked up a fuss about this but bigquery was fine 🤔 

Yeh I think we could handle the case where a column exists in one table but not another much better than we do (right now we just see an error like this, but it would be more helpful to say something like "description exists in the expected results but not in the actual output"). 

Beyond the scope of this PR though so I guess i'll just add a `description` to the raw input, which doesn't match the one in the expected results, so we'll error if we ever compare them. Let me know if you can think of a better workaround tho 😄

Reply:
This error seem to be unsolved for Postgres? #84